yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtRoot.kt

```diff
@@ -46,7 +47,7 @@ internal class CrdtRoot(val rootObject: CrdtObject) {
         val subPaths = mutableListOf<String>()
         while (pair.parent != null) {
```

Comment:
how about chaning like below to remove ugly `!!`s :)
```suggestion
        while (true) {
            val parent = pair.parent ?: break
```